### ISSUE TITLE
Reject: A multicenter study of automatic segmentation-based multimodal fusion integratin

### ISSUE THREAD

**Issue by Mantrainment** [Mantrainment/neuroscience_newsletter#126]
A multicenter study of automatic segmentation-based multimodal fusion integrating radiomics, deep learning, and clinical parameters for prostate cancer detection.
---
To develop and validate an interpretable machine learning model integrating radiomics, deep learning (DL), and clinical features based on automated MRI segmentation for detecting prostate cancer (PCa). This retrospective multicenter study included 433 prostate patients. The internal cohort comprised 346 patients, who were randomly divided into a training set (n = 242) and an internal validation set (n = 104) at a 7:3 ratio. Automated prostate segmentation was performed on T2-weighted imaging and apparent diffusion coefficient maps using TotalSegmentator. Radiomics features were extracted and selected via mutual information, mRMR, LASSO, and Pearson correlation analysis. The DL labels were derived from a DenseNet-121 convolutional neural network. Using the eXtreme Gradient Boosting (XGBoost) algorithm, we constructed four types of prediction models: clinical models, radiomics models, DL models, and combined models integrating all three feature types (clinical, radiomics, and DL features). Model performance was evaluated using area under the curve (AUC) and related metrics. An external test set of 87 patients was used to validate the predictive performance of each model. Finally, SHapley Additive exPlanations (SHAP) were applied to enhance model interpretability and quantify the impact of individual features. The combined model integrating radiomics, DL, and clinical features achieved the highest AUC of 0.902 (95% CI 0.841-0.962) in the external test set, outperforming individual models. SHAP analysis revealed prostate-specific antigen density and DL label as dominant predictors and provided transparent global and local interpretations of model decisions. The combined model integrating clinical, radiomics, and DL features based on automated MRI segmentation, achieved high accuracy and promising clinical utility in PCa detection.
